### ISSUE TITLE
HE Web Interface and Container Inserts/Extracts

### ISSUE THREAD

**Issue by bmlzootown** [bob7l/HawkReloaded#61]
I realize the web interface hasn't been updated in quite a while (and prior to now, it hasn't really needed one). As it currently stands, however, one can no longer search for transactions due to the switch to inserts and extracts. Will there be an update to reflect this change in the near future?

**Comment by bob7l** [comment]
I'm confused.. Are you saying you cannot search for extracts/inserts in the web interface, or using the HawkEye plugin? 

And it will be done in a few weeks or so hopefully. I've been making sure the actual plugin is stable before i work on it. I'll be pretty busy for at least a month so we'll have to wait and see. 

If anyone wants to take this task under their wing it shouldn't be difficult. It's just basic PHP and the inventory transactions are saved in a JSON-LIKE format i created just or serialization.

**Comment by bmlzootown** [comment]
Indeed, I meant the web interface. I also went ahead and updated my own copy. It's not perfect, by far, but it works. If I had thought about it beforehand, I would have organized it a bit better and submitted a pull request.
